packages/mongodb-chatbot-ui/src/Chatbot.tsx

```diff
@@ -8,26 +8,35 @@ import { type User } from "./useUser";
 import { ChatbotProvider } from "./ChatbotProvider";
 
 export type ChatbotProps = {
+  children: React.ReactElement | React.ReactElement[];
   darkMode?: boolean;
-  tck?: string;
+  isExperimental?: boolean;
+  maxInputCharacters?: number;
+  name?: string;
   serverBaseUrl?: string;
   shouldStream?: boolean;
+  tck?: string;
   user?: User;
-  children: React.ReactElement | React.ReactElement[];
 };
 
 export function Chatbot({
   children,
   serverBaseUrl,
   shouldStream,
   user,
+  name,
   ...props
 }: ChatbotProps) {
   const { darkMode } = useDarkMode(props.darkMode);
 
+  // The server currently uses a max of 300 input characters, so we'll default to that here
+  const maxInputCharacters = props.maxInputCharacters ?? 300;
```

Comment:
```suggestion
  const DEFAULT_MAX_INPUT_CHARACTERS = 300;
  const maxInputCharacters = props.maxInputCharacters ?? DEFAULT_MAX_INPUT_CHARACTERS;
```

to avoid having that comment which could go stale at any point